README.md

```diff
@@ -211,6 +211,61 @@ If you also want to compile C/C++ to WebAssembly (and not just asm.js to WebAsse
 
    CMake generates a project named "ALL_BUILD.vcxproj" for conveniently building all the projects.
 
+### Building binaryen.js
+
```

Comment:
This seems like the wrong place to be documenting this.

Building binaryen.js is just like building any other C/C++ project with cmake.  There is nothing binaryen-specific about these docs.  If we have any docs at all here it should just be a link to "hot to use emcmake" to build cmake projects with emscripten.

Duplicating docs on "how to install emsdk" here seems like a bad idea.

Are going to go to every C++ project at ask them to document how to do that emscripten build in their upstream README?